api/channel.go

```diff
@@ -159,9 +155,13 @@ type ModifyChannelData struct {
 	//
 	// Channel Types: All
 	Permissions *[]discord.Overwrite `json:"permission_overwrites,omitempty"`
+	// Name is the 2-100 character channel name.
+	//
+	// Channel Types: All
+	Name string `json:"name,omitempty"`
 	// CategoryID is the id of the new parent category for a channel.
 	// Channel Types: Text, News, Store, Voice
```

Comment:
```suggestion
	// CategoryID is the id of the new parent category for a channel.
	//
	// Channel Types: Text, News, Store, Voice
```

Insert an empty comment in-between those lines.